ionic/src/app/add-employee/add-employee.component.ts

```diff
@@ -7,9 +8,19 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AddEmployeeComponent implements OnInit {
 
-  constructor() { }
+  constructor(
+    private deviceService: DeviceService,
+  ) { }
 
   ngOnInit() {
   }
-
+  async setSwitchNamee(d: Device) {
+    this.deviceService.sendMessageToSocket({
+      type: "device_set_add_employee",
```

Comment:
this is function setSwitchName, why is type add employee

Reply:
in add employee not need setSwitchName thats why ill romove form here
and for add employee i don't have any api or emp_id thats why i remove this function from now when i get then ill write function again